Readme.md

```diff
@@ -3,57 +3,134 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T190813)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# Spreadsheet for ASP.NET MVC - How to save/load documents to/from a database
 
-* [HomeController.cs](./CS/DXWebApplication23/Controllers/HomeController.cs) (VB: [HomeController.vb](./VB/DXWebApplication23/Controllers/HomeController.vb))
-* [DataHelper.cs](./CS/DXWebApplication23/Models/DataHelper.cs) (VB: [DataHelper.vb](./VB/DXWebApplication23/Models/DataHelper.vb))
-* [Index.cshtml](./CS/DXWebApplication23/Views/Home/Index.cshtml)
-* **[SpreadsheetPartial.cshtml](./CS/DXWebApplication23/Views/Home/SpreadsheetPartial.cshtml)**
-<!-- default file list end -->
-# Spreadsheet - How to save and load documents from a database
-
-## Steps to implement:
-1) Create a helper class that returns the Spreadsheet settings. Make sure that you specify the **Name** and the **CallbackRouteValues** properties:
-```cs
- public static class SpreadsheetSettingsHelper {
+This example demonstrates how to configure the [Spreadsheet extension](https://docs.devexpress.com/AspNetMvc/17113/components/spreadsheet) to save/load documents to/from a database.
+
+![Connect Spreadsheet to Database](connect-spreadsheet-to-database.png)
+
+## Overview
+
+Follow the steps below to configure the Spreadsheet extension to work with a database:
+
+1. Create a class that models a spreadsheet document. This class should be able to store a document's identifier, format, and content:
```

Comment:
```suggestion
1. Create a class that models a spreadsheet document. This class should be able to store document identifiers, formatting, and content:
```

Reply:
I'm not sure we could replace the "format" word with the "formatting" word. In this sentence I'm talking about document file formats (such as DOCX, PDF, TXT). Document formatting term refers to the way a document looks and addresses things like margins, spacing, font size.

Can we say something like the following?
```suggestion
1. Create a class that models a spreadsheet document. This class should be able to store document identifiers, file formats, and content:
```

Reply:
this works